quarkus/admin/distribution/NOTICE

```diff
@@ -15,358 +15,11 @@ This binary artifact bundles the following projects with NOTICE:
 
 Group: info.picocli Name: picocli Name: 4.7.7
 
-NOTICE:
+Relevant information from NOTICE:
 | This project includes one or more documentation files from OpenJDK, licensed under GPL v2 with Classpath Exception.
-| 
+|
 | These files are included in the source distributions, not in the binary distributions of this project.
```

Comment:
This is kind of more confusing for me, why are we including this notice if it isn't relevant?  The GPL files are included in the source distro of piccoli which isn't used by Polaris?

Reply:
Yes, we are using picocli jar file that doesn't include the OpenJDK documentation files (GPL).  So GPL only applies to picocli source distribution (not jar).

My proposal here is:
1. to include the first partt of the picocli `NOTICE` describing that GPL files are only in the picocli source disrtibution
2. remove the GPL license as it doesn't apply

Reply:
Sounds reasonable to me.

Reply:
@RussellSpitzer is it still confusing to you ? Or are we good to move forward here ?

Reply:
OK, I don't want to hold this change longer in order to move forward on RC3.